From b483873253558513d0478dfc1e0d65ea81dc9df1 Mon Sep 17 00:00:00 2001
Message-ID: <b483873253558513d0478dfc1e0d65ea81dc9df1.1771966444.git.lorenzo@kernel.org>
In-Reply-To: <2cd15ef3379025ea1ae1f6c36c70621aa578c270.1771966444.git.lorenzo@kernel.org>
References: <2cd15ef3379025ea1ae1f6c36c70621aa578c270.1771966444.git.lorenzo@kernel.org>
From: Lorenzo Bianconi <lorenzo@kernel.org>
Date: Fri, 19 Dec 2025 23:12:32 +0100
Subject: [bpf-next 11/16] net: airoha: Rename airhoha_set_gdm2_loopback in
 airhoha_enable_gdm2_loopback

This is a preliminary patch in order to allow the user to select if the
configured device will be used as hw lan or wan.

Signed-off-by: Lorenzo Bianconi <lorenzo@kernel.org>
---
 drivers/net/ethernet/airoha/airoha_eth.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/ethernet/airoha/airoha_eth.c
+++ b/drivers/net/ethernet/airoha/airoha_eth.c
@@ -1720,7 +1720,7 @@ static int airoha_dev_set_macaddr(struct
 	return 0;
 }
 
-static int airhoha_set_gdm2_loopback(struct airoha_gdm_dev *dev)
+static int airhoha_enable_gdm2_loopback(struct airoha_gdm_dev *dev)
 {
 	struct airoha_gdm_port *port = dev->port;
 	struct airoha_eth *eth = dev->eth;
@@ -1795,7 +1795,7 @@ static int airoha_dev_init(struct net_de
 		if (!eth->ports[1]) {
 			int err;
 
-			err = airhoha_set_gdm2_loopback(dev);
+			err = airhoha_enable_gdm2_loopback(dev);
 			if (err)
 				return err;
 		}
